python-package-prompt-guard/src/promptguard/promptguard_service.py

```diff
@@ -16,7 +16,7 @@
 @dataclass
 class SanitizeResponse:
     sanitized_text: str
-    secure_context: bytes
+    secure_context: str
```

Comment:
While the go server technically returns a byte object, it is converted to a string by the JSONification before the python code sees it.

Reply:
Can you add this as a comment?